community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/BoltStateMachine.java

```diff
@@ -326,6 +326,8 @@ public State init( BoltStateMachine machine, String userAgent,
                             {
                                 machine.ctx.addMetadata( "credentials_expired", true );
                             }
+                            machine.ctx.addMetadata( "neo4j_version", machine.spi.version() );
```

Comment:
Can we alter this to `{ ... "server": "Neo4j/3.1.0" ... }` to both match the format of the user agent strings and to mirror HTTP? This also prevents coupling the protocol specification so tightly to the product.